test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -495,8 +492,7 @@ public void testIncludeAllColumnsAsMetadata_mcSet() throws Exception {
   @Test
   public void testIncludeAllColumnsAsMetadataFalse_mcBlank() throws Exception {
     Map<String, String> moreEntries = new HashMap<String, String>();
-    moreEntries.put("adaptor.namespace", "Default");
-    moreEntries.put("db.modeOfOperation", "rowToText");
+    moreEntries.put("db.includeAllColumnsAsMetadata", "false");
```

Comment:
Add a TODO that we should add a "Default_mcBlank" variation once we're using initConfig.